app/lib/queues/populatePracticeQueue.js

```diff
@@ -83,7 +90,26 @@ function queueSyndicationIds(q) {
     }
   });
 }
+function queueErroredIds(q) {
+  const failedIds = gpStore.getErorredIds();
+  totalRetries = failedIds.length;
+  gpStore.clearFailedIds(failedIds);
+  failedIds.forEach((id) => {
+    if (id) {
+      q.push({ id }, () => log.info(`${id} done`));
+    } else {
+      log.error('Undefined ID in syndication list');
+    }
+  });
+}
 
+function startRetryQueue(workers, drain) {
```

Comment:
I think there's enough significant differences between the two functions (i.e. they update a different count, run a different function on each item, and call a different method to add items to the queue) to warrant them being separate functions.